commands/core/site_install.drush.inc

```diff
@@ -21,6 +21,7 @@ function site_install_drush_command() {
         'description' => 'Password for the "db-su" account. Optional.',
         'example-value' => 'pass',
       ),
+      'db-drop-existing-tables' => 'Defaults to TRUE. Remove all tables in the target database before creating Drupal tables.',
```

Comment:
Lets say move the 'Defaults to TRUE' to the end. Also, make the value an array and add key 'example-value' with value of 0 so usage is clearer. Thanks.